### PR TITLE
92 define different environment configuration files

### PR DESCRIPTION
**Pull request by estefi-prieto**

### CODE REVIEW

File: docs/Guide-to-configure-your-data-source.md [hunk 1]
```diff
@@ -33,17 +33,23 @@
...
 ![06](./images/Guide-to-configure-your-data-source/06s.png)
 
-  * And your API, should have the following url:
+* Create the enviroments/development.json and enviroments/production.json files.
+
+* In both files, set your appropiate API url, like as:
+
+```JSON
+{
+"API_URL": "your-lan-ip"
```
Comment (sgobotta) [inline docs/Guide-to-configure-your-data-source.md]: There we have missed some indentation.

File: docs/Guide-to-configure-your-data-source.md [hunk 2]
```diff
@@ -33,17 +33,23 @@
...
+}
+```
+
+  * Your API should have the following url:
```
Comment (sgobotta) [inline docs/Guide-to-configure-your-data-source.md]: I'd rename url for services.

File: .gitignore [hunk 3]
```diff
@@ -7,3 +7,5 @@ node_modules
 # NativeScript application
 hooks
 platforms
+
+enviroments
```
Comment (sgobotta) [inline .gitignore]: It'd be environments

**Review by sgobotta — CHANGES_REQUESTED** [review]

File: docs/Guide-to-configure-your-data-source.md (hunk 2 above)
Comment (estefi-prieto) [reply docs/Guide-to-configure-your-data-source.md]: I'm agree

File: environments/backend.json [hunk 4]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (sgobotta) [inline environments/backend.json:1]: This file can be deleted **if** we are not using it.

**Review by sgobotta — APPROVED** [review]

**Pull request merged**